setup.py

```diff
@@ -12,6 +12,6 @@
     license="GPLv2",
 
     # Requirements
-    install_requires=["numpy>1.6", "six", "enum34"],
+    install_requires=["numpy>1.6", "six", "enum34", "sentinel"],
```

Comment:
Oops! That has been required by place and route too! By the way why do we have the requirements*.txt files when this exists?